src/configuration/syncSettings.js

```diff
@@ -1,4 +1,5 @@
 export default function syncSettings(settings) {
+    settings.id_cols = settings.nestings.filter(d => d.default === true).map(f => f.value_col);
```

Comment:
Sweet array!  In the edge case that all four nestings have `d.default === true` maybe `.slice(0,3)` after the `.filter()`.